bundle/next/eclipse-che/manifests/org.eclipse.che_checlusters.yaml

```diff
@@ -14,7 +14,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.7.0
```

Comment:
@mkuznyetsov I think something went wrong here? controller-gen got downgraded and the diff is much larger than expected for this PR. Maybe your local version of controller-gen/kubebuilder is outdated?